python/shotgun_data/shotgun_data_retriever.py

```diff
@@ -1068,25 +1068,32 @@ def _on_task_completed(self, task_id, group, result):
             return
 
         action = result.get("action")
-        if action in ["find", "find_one", "create", "delete", "update"]:
-            self.work_completed.emit(str(task_id), action, {"sg":result["sg_result"]})
-        elif action == "nav_expand":
-            self.work_completed.emit(str(task_id), "nav_expand", {"nav":result["nav_result"]})
+        if action in ["find", "find_one", "create", "delete", "update", "nav_expand"]:
+            self.work_completed.emit(
+                str(task_id),
+                action,
+                {"sg": result["sg_result"]}
+            )
         elif action == "schema":
-            self.work_completed.emit(str(task_id), "schema", {"fields":result["fields"], "types":result["types"]})
+            self.work_completed.emit(
+                str(task_id),
+                "schema",
+                {"fields": result["fields"], "types": result["types"]}
+            )
         elif action == "method":
-            self.work_completed.emit(str(task_id), "method", {"return_value":result["result"]})
+            self.work_completed.emit(
+                str(task_id),
+                "method",
+                {"return_value": result["result"]}
+            )
         elif action == "check_thumbnail":
             path = result.get("thumb_path", "")
             if path:
                 # check found a thumbnail!
                 self.work_completed.emit(
                     str(task_id),
-                    "find",
```

Comment:
Yes - this is a cut n paste typo - @jfboismenu you ok with it changing to its "correct" value?